config/datatypes_conf.xml.sample

```diff
@@ -412,6 +412,12 @@
     <datatype extension="xmfa" type="galaxy.datatypes.msa:MauveXmfa" display_in_upload="True" />
 
     <datatype extension="RData" type="galaxy.datatypes.binary:RData" display_in_upload="true" description="Stored data from an R session"/>
+    <datatype extension="oxlicg" type="galaxy.datatypes.binary:OxliCountGraph" mimetype="application/octet-stream" display_in_upload="true"/>
```

Comment:
Is `mimetype="application/octet-stream"` here
necessary given that the supertype `Binary`'s `.get_mime()` returns `application/octet-stream`?